src/rebar3_hex_docs.erl

```diff
@@ -146,7 +146,7 @@ do_revert(App, State, Repo, Vsn) ->
     Name = rebar_utils:to_list(rebar_app_info:name(App)),
     PkgName = rebar_utils:to_list(proplists:get_value(pkg_name, AppDetails, Name)),
     case rebar3_hex_client:delete_docs(Config, rebar_utils:to_binary(PkgName), rebar_utils:to_binary(Vsn)) of
-        {modified, _Body} ->
+        {ok, _Body} ->
```

Comment:
`Body`'ll be, most likely `nil`. This was confirmed between me and @starbelly.

Reply:
This may get fixed in #232. Because it introduces a hex_http adapter which uses hex_http behavior and because I didn't want to turn off dialyzer, etc. I opted to fix it up there. We'll see what the morning brings.

Reply:
Sure, I'll rebase and retry no worries. Thanks.